ztp/extra-manifests-builder/99-one-shot-time-sync/sync-time-once.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+SYNC_ATTEMPT_TIMEOUT_SEC=300
+
+/usr/sbin/chronyd -n -f /etc/chrony.conf -q -t $SYNC_ATTEMPT_TIMEOUT_SEC
+if [[ $? -ne 0 ]]; then
+  echo "failed to achieve time sync in ${SYNC_ATTEMPT_TIMEOUT_SEC}s"
```

Comment:
Does the chronyd executable not already say something like this if it fails?

Reply:
Actually, if chronyd has good enough output on its own, we could eliminate this script entirely and just call chronyd directly in the systemd unit, which would make this a lot simpler!

Reply:
No, unfortunately not. It just says it exited.
For example:
```bash
# /usr/sbin/chronyd -n -f /etc/chrony.conf -q -t 1
2023-03-15T12:46:14Z chronyd version 4.1 starting (+CMDMON +NTP +REFCLOCK +RTC +PRIVDROP +SCFILTER +SIGND +ASYNCDNS +NTS +SECHASH +IPV6 +DEBUG)
2023-03-15T12:46:14Z Frequency 23.213 +/- 0.005 ppm read from /var/lib/chrony/drift
2023-03-15T12:46:15Z chronyd exiting
```
Same string is printed when it succeeds and exits due to -q flag